crawler/src/main/java/nl/fhict/happynews/crawler/crawler/Crawler.java

```diff
@@ -10,16 +10,20 @@
 import org.springframework.dao.DuplicateKeyException;
 import org.springframework.data.domain.Example;
 import org.springframework.data.domain.ExampleMatcher;
+import org.springframework.stereotype.Service;
 
 import java.util.List;
 
 /**
  * Created by Tobi on 27-Mar-17.
+ * Classes extending crawler class also need the @Service annotation
+ *
  */
+@Service
 public abstract class Crawler<T> {
 
     @Autowired
-    private PostRepository postRepository;
+    protected PostRepository postRepository;
```

Comment:
changed this for testing but it didn't solve the problem, i changed it back to private now